subprojects/model-core/src/test/groovy/org/gradle/internal/properties/annotations/NestedBeanAnnotationHandlerTest.groovy

```diff
@@ -66,51 +63,13 @@ class NestedBeanAnnotationHandlerTest extends Specification {
 
     def "absent optional nested property is ignored"() {
         when:
-        handler.visitPropertyValue("name", value, propertyMetadata, propertyVisitor, context)
+        handler.visitPropertyValue("name", value, propertyMetadata, propertyVisitor)
 
         then:
         1 * value.call() >> null
         1 * propertyMetadata.isAnnotationPresent(TestOptional) >> true
         0 * _
     }
-
-    def "exception thrown by nested property is propagated"() {
```

Comment:
💭 This is checked in `NestedInputIntegrationTest#execution fails when a nested property throws an exception`